.thoth.yaml

```diff
@@ -8,7 +8,7 @@ runtime_environments:
       name: ubi
       version: "8"
     python_version: "3.6"
-    recommendation_type: stable
+    recommendation_type: security
```

Comment:
To demo the difference, would it make sense to put this under a directory (e.g. `security`) and have another directory aside (e.g. `latest`) so show the difference in output?

Reply:
I'm pulling this to a different branch. I could create another branch that does a `latest` advise